### PR TITLE
Users can now update data feed results from the UI

### PR DESCRIPTION
**Pull request by emaG3m**

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -28,6 +28,14 @@ export const fetchDataFeedLatestResult = ({ dataFeedAddress }) => dispatch => {
   )
 }
 
+export const logDataFeedResult = ({ address }) => dispatch => {
```
Comment (mikec) [inline app/actions/index.js]: might want to name the param `dataFeedAddress`

File: app/client/index.js [hunk 2]
```diff
@@ -31,6 +20,26 @@ export const removeDataFeed = async ({ address }) => {
...
+  const currentResult = await tokenPriceDataFeed.currentPrice()
+  const lastUpdated = await tokenPriceDataFeed.lastUpdated()
+
+  return {
+    currentResult,
+    lastUpdated
+  }
+}
+
+export const logDataFeedResult = async ({ address }) => {
```
Comment (mikec) [inline app/client/index.js]: same here, `dataFeedAddress`

File: app/containers/DataFeedListContainer.js [hunk 3]
```diff
@@ -10,6 +10,10 @@ const mapStateToProps = (state) => ({
 const mapDispatchToProps = dispatch => ({
   handleRemove: ({ address }) => {
     dispatch(removeDataFeed({ address }))
+  },
+  handleUpdate: ({ address }) => {
+    console.log('here1')
```
Comment (mikec) [inline app/containers/DataFeedListContainer.js]: debugging statement hangin' around

File: scripts/test1234.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+const abi = require('../app/client/TokenPriceDataFeed.abi')
```
Comment (mikec) [inline scripts/test1234.js]: is this file checked in accidentally?

File: contracts/IDataFeed.sol [hunk 5]
```diff
@@ -2,8 +2,9 @@ pragma solidity >=0.4.24;
 
 import "tidbit/contracts/DataFeedOracles/DataFeedOracleBase.sol";
 
-interface ITidbitDataFeedOracle {
+interface IDataFeed {
   function setResult(DataFeedOracleBase[] memory _dataFeeds) public;
```
Comment (mikec) [inline contracts/IDataFeed.sol]: does this need both `setResult` and `logResult`? maybe `setResult` can be removed now?

Comment (emaG3m) [reply contracts/IDataFeed.sol]: yah that's true. It also feels a big weird to have the Interface in this repo, that datafeeds outside the repo need to follow. Fine for now, but there's something about this that feels a bit....off to me right now

File: scripts/test1234.js (hunk 4 above)
Comment (emaG3m) [reply scripts/test1234.js]: yup

File: app/client/index.js [hunk 6]
```diff
@@ -31,6 +20,27 @@ export const removeDataFeed = async ({ address }) => {
...
+  const lastUpdated = await tokenPriceDataFeed.lastUpdated()
+
+  return {
+    currentResult,
+    lastUpdated
+  }
+}
+
+export const logDataFeedResult = async ({ dataFeedAddress }) => {
+  console.log('logDataFEedResult ', dataFeedAddress)
```
Comment (mikec) [inline app/client/index.js]: looks like a debugging statement hanging around

**Comment by mikec** [comment]
@emaG3m just one console.log that should be removed, looks good 👍 👍

**Pull request merged**